docs/setup/pip.md

```diff
@@ -16,7 +16,7 @@ We support :simple-python: **Python from 3.8 to 3.12**.
     wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/10.2.0/khiops-core_10.2.0-1-${CODENAME}.amd64.deb" && \
     sudo dpkg -i "$TEMP_DEB" || sudo apt-get -f -y install && \
     rm -f $TEMP_DEB && \
-    pip install 'https://github.com/KhiopsML/khiops-python/releases/download/10.2.0.0/khiops-10.2.0.0.tar.gz'
+    pip install 'https://github.com/KhiopsML/khiops-python/releases/download/10.2.0.1/khiops-10.2.0.1.tar.gz'
```

Comment:
Change "10.2.0.1" to "10.2.1.0" (indeed, it is 10.2.1.0 which contains the Rocky 8 fix).

Reply:
Done.